docs/sql-reference/sql-statements/data-types/JSON.md

```diff
@@ -101,18 +101,15 @@ Example 2: Filter the data of the JSON column to retrieve the data that meets th
 
 ```Plain%20Text
 mysql> select * from tj where j->'a' = parse_json('1');
-+------+---------------------+
-|   id |         j           |
-+------+---------------------+
-|   1  | {"a": 1, "b": true} |
-+------+---------------------+
+Empty set (0.05 sec)
```

Comment:
this example is used to show the ability of json predicate, why remove it ?

Reply:
I'm not removing it.   I'm saying it's wrong and replacing it with the correct syntax.